drivers/timer/stm32_lptim_timer.c

```diff
@@ -46,6 +46,10 @@ static const struct stm32_pclken lptim_clk[] = STM32_DT_INST_CLOCKS(0);
 
 static const struct device *const clk_ctrl = DEVICE_DT_GET(STM32_CLOCK_CONTROL_NODE);
 
+/* For tick accuracy, a specific tick to freq ratio is expected */
+/* This check assumes LSI@32KHz or LSE@32768Hz */
+#define LPTIM_CLOCK_RATIO DT_PROP(DT_DRV_INST(0), st_prescaler)
```

Comment:
Since you're mentioning `freq ratio` just before, naming the C value equivalent of `st-prescaler` as `LPTIM_CLOCK_RATIO` is a bit disturbing.
What about `LPTIM_PRESCALER`? Now that there is actually an important ratio notion in the driver, it may help the reader.